webdriver-spec.html

```diff
@@ -791,6 +791,11 @@
  has an associated list of <dfn>remote end steps</dfn>.
  This provides the sequence of actions that a <a>remote end</a> takes
  when it receives a particular <a>command</a>.
+ All <dfn data-lt="RES-arguments"><a>remote end steps</a>
```

Comment:
Actually I’m ambivalent towards https://github.com/w3c/webdriver/pull/754#discussion_r102746154. Maybe it is better for clarity.